config/eslint/frontend/.eslintrc.js

```diff
@@ -0,0 +1,38 @@
+module.exports = {
+  env: {
+    es6: true,
+    browser: "true",
+    "jest/globals": true
+  },
+  extends: [
+    "plugin:styled-components-a11y/recommended",
+    "plugin:prettier/recommended",
+    "react-app",
+    "eslint:recommended",
+    "plugin:react/recommended",
+    "cypress"
+  ],
+  globals: {},
+  plugins: ["react", "react-hooks", "@emotion", "jest"],
+  rules: {
+    "react/jsx-filename-extension": 0,
+    "react/jsx-one-expression-per-line": 0,
+    "react/jsx-props-no-spreading": 0,
+    "react/destructuring-assignment": [2, "always"],
+    "react/no-deprecated": 2,
+    "react/no-unsafe": 2,
+    "react/no-unused-state": 2,
+    "react/jsx-pascal-case": 2,
+    "react/react-in-jsx-scope": 0,
+    "react-hooks/exhaustive-deps": 0,
+    "@emotion/syntax-preference": [2, "object"],
+    "jsx-a11y/no-static-element-interactions": 0,
```

Comment:
Should have mentioned earlier, I think this is something we should consider turning on for accessibility purposes

Reply:
another rule that is related that might be worth adding https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/main/docs/rules/click-events-have-key-events.md

Reply:
> another rule that is related that might be worth adding [https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/main/docs/rules/click-events-have-key-events.md](https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/main/docs/rules/click-events-have-key-events.md?rgh-link-date=2022-05-17T15%3A58%3A10Z)

I think the move here is to include the `jsx-ally` plugin, I think it'll have that rule (and others) on by default?